internal/app/app.go

```diff
@@ -38,17 +38,21 @@ type App struct {
 }
 
 // New returns returns a new instance of the flipflop app
-func New(storeKind model.StoreKind, cfgFile, loglevel string, profiling bool) (*App, <-chan os.Signal, error) {
+func New(cfgFile, loglevel string, profiling bool) (*App, <-chan os.Signal, error) {
 	app := &App{
 		v:      viper.New(),
 		Config: &Configuration{},
 		Logger: logrus.New(),
```

Comment:
Given that config likely has a logging level in it, maybe defer this to `LoadConfiguration`? That would also let us bury the set for `app.Logger.Level` in there too?

Reply:
mhmm, yea ill look at that.